test/envoye2e/driver/envoy.go

```diff
@@ -145,3 +156,60 @@ func getAdminPort(bootstrap string) (uint32, error) {
 	}
 	return port.PortValue, nil
 }
+
+// downloads env based on the given branch name. Return address of downloaded envoy.
```

Comment:
Nit: returns the “location” of the envoy.

Reply:
Done.